core/spec.md

```diff
@@ -491,45 +504,158 @@ Content-Length: nnnn
 }
 ```
 
+##### Updating the Registry Model
```

Comment:
earlier today I was thinking something similar for the Group.format field since updating it could make all resources in the Group non-compliant. I think in general these actions need to be defined so people can build-up a Registry. One option is to make it so that these fields can only be updated if the result is a compliant Registry - how that's determined is out of scope for the spec. So, for "group's schema" it could mean the user would need to delete all instances of a Group before its schema is updated.  Maybe?  Or delete the Groups schema URL, update all Groups to the new schema, then when done update the Registry's group.schema URL ??? None seem nice, but I'm more worried about trying to define an atomic action that does it all at once.